playwright/tests/liveblog-inline-slots.spec.ts

```diff
@@ -5,49 +5,45 @@ import { cmpAcceptAll } from '../lib/cmp';
 import { loadPage } from '../lib/load-page';
 import { countLiveblogInlineSlots } from '../lib/util';
 
-const blogPages = blogs.filter(
-	(page) =>
-		'expectedMinInlineSlotsOnDesktop' in page &&
-		'expectedMinInlineSlotsOnMobile' in page,
-);
+const blogPages = blogs.filter((page) => 'expectedMinInlineSlots' in page);
 
 test.describe.serial('A minimum number of ad slots load', () => {
 	blogPages.forEach(({ path, expectedMinInlineSlots }) => {
 		breakpoints.forEach(({ breakpoint, width, height }) => {
 			const isMobile = breakpoint === 'mobile';
 			const expectedMinSlotsOnPage =
-				expectedMinInlineSlots?.[
+				expectedMinInlineSlots[
 					breakpoint as keyof typeof expectedMinInlineSlots
 				];
 
-			if (expectedMinSlotsOnPage) {
```

Comment:
This was never true due to the filter on line 8! So these tests were not running